flight/PiOS/inc/pios_can.h

```diff
@@ -35,6 +35,7 @@
 
 //! The set of CAN messages
 enum pios_can_messages {
+	PIOS_CAN_ERROR = -1,
```

Comment:
In general gcc lets you assign and implicit cast unsigned and signed types without warnings-- it just gets mad about comparisons.  (Unfortunately the alternative would be way too chatty)